src/main/java/org/opentripplanner/apis/common/json/GraphQLResponseSerializer.java

```diff
@@ -1,4 +1,4 @@
-package org.opentripplanner.api.json;
+package org.opentripplanner.apis.common.json;
```

Comment:
Move to `org.opentripplanner.framework.graphql`.

There is no OTP dependencies in this so this is just an extension to the graphQl framework; Hence, belong in framework.

Reply:
8be001d3d0d5f9fe8c234a4ae0ee95f923fc7544